src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -93,10 +93,41 @@
 				</div>
 				<div class="row">
 					<div class="columns small-12 large-expand">
-						Lend button
+						<action-button
+							class="expandable-loan-card-action-button"
+							:loan-id="loan.id"
+							:loan="loan"
+							:items-in-basket="itemsInBasket"
+							:is-lent-to="loan.userProperties.lentTo"
+							:is-funded="isFunded"
+							:is-selected-by-another="isSelectedByAnother"
+							:is-simple-lend-button="false"
+
+							@click.native="trackInteraction({
+								interactionType: 'addToBasket',
+								interactionElement: 'Lend25'
+							})"
+
+							@add-to-basket="$emit('add-to-basket', $event)"
+						/>
 					</div>
-					<div class="matching-text columns small-12 large-4">
-						2x matching
+					<div class="columns small-12 large-4 show-for-large matching-text-wrap">
```

Comment:
If we delete `show-for-large` we can remove the duplicate matching text. We can change `small-12` to `medium-12`, then we can just style it to center (and maybe add a `padding-top`, might not be necessary) on the medium breakpoint down.